src/Microsoft.AspNet.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -420,7 +420,10 @@ internal static PropertyValidationInfo GetPropertyValidationInfo(ModelBindingCon
 
                 bindingContext.ModelState.TryAddModelError(
                     modelStateKey,
-                    Resources.FormatModelBinding_MissingBindRequiredMember(propertyName));
+                    string.Format(
+                        CultureInfo.CurrentCulture,
+                        propertyExplorer.Metadata.ValidationErrorMessages.MissingBindRequiredValueResource(),
+                        propertyName));
```

Comment:
Tempting to make the resource `Func`s take the arguments and avoid these shenanigans. Any objection?